library/src/main/java/com/novoda/downloadmanager/Migration.java

```diff
@@ -74,14 +74,15 @@ public String toString() {
     }
 
     static class FileMetadata {
+
         private final String originalFileLocation;
         private final FileSize fileSize;
-        private final String uri;
+        private final String originalNetworkAddress;
```

Comment:
Renamed these because I had to keep jumping backwards and forwards to find out what each of them was doing 😄